cmd/mps-control-daemon/mount/mount-shm.go

```diff
@@ -17,26 +17,46 @@
 package mount
 
 import (
+	"bufio"
 	"fmt"
 	"os"
 	"os/exec"
+	"strconv"
+	"strings"
 
 	"github.com/urfave/cli/v2"
+	"k8s.io/klog/v2"
 	"k8s.io/mount-utils"
 )
 
+type options struct {
+	shmSize string
+}
+
 // NewCommand constructs a mount command.
 func NewCommand() *cli.Command {
-	// Create the 'generate-cdi' command
-	return &cli.Command{
-		Name:   "mount-shm",
-		Usage:  "Set up the /dev/shm mount required by the MPS daemon",
-		Action: mountShm,
+	opts := options{}
+
+	c := cli.Command{
+		Name:  "mount-shm",
+		Usage: "Set up the /dev/shm mount required by the MPS daemon",
+		Action: func(ctx *cli.Context) error {
+			return mountShm(ctx, &opts)
+		},
```

Comment:
Can we have this component directly use the plugins `config` file and avoid introducing flags?

Reply:
I will remove the flags. I will work in the use of the config as part of #699 if we determine that this needs to be set.

One thing to note is that allowing this option to be set complicates the syncronisation between the various components considerably since this container would also need to be restarted on a config change.